ipa.go

```diff
@@ -47,6 +49,15 @@ func toFr(fr *Fr, p *Point) {
 	fr.SetBytesLE(bytes[:])
 }
 
+func toFrMultiple(ps []*Point) []Fr {
+	pss := banderwagon.MultiMapToBaseFieldBytes(ps)
+	frs := make([]Fr, len(ps))
+	for i := range pss {
+		frs[i].SetBytesLE(pss[i][:])
```

Comment:
As mentioned on discord, it seems very wasteful to get the bytes of the field element in `MultiMapToBaseFieldBytes`, and immediately use these bytes to call `SetBytes`.

Reply:
Agree! Avoiding `Element -> bytes -> from bytes -> Element` would be great.